metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -82,9 +97,19 @@
     "box_plot": ChartTypeClass.BAR,
 }
 
+# Set Mapping to Postgres by default
+SUPERSET_FIELD_TYPE_MAPPINGS = POSTGRES_TYPES_MAP
```

Comment:
we can use the newly added `resolve_sql_type` method instead of this

Reply:
I've added this mapping logic to `get_platform_from_database_id` -> Platform is tied to the database